packages/cli-tools/src/launchPackager.command

```diff
@@ -1,3 +1,5 @@
+#!/bin/sh
```

Comment:
I don't believe `BASH_SOURCE` is going to be set in `sh`, this will probably need to be `bash`

Reply:
Ah, you're right 👍 Fixed.